src/components/common/Footer.vue

```diff
@@ -6,8 +6,10 @@
           v-col(cols="12" md="6").text-center
             v-layout(row)
               v-container
-                span ABOUT US
+                p ABOUT US
                 p Student Gymkhana IITJ
+                p
+                  a(href="https://devlup-labs.github.io/#/" class="white--text" style="text-decoration: none;") Devlup Labs
```

Comment:
Can you remove this link and add it to the `v-footer` and change `by Students' Gymkhana WebD Team` to `Devlup Labs`